### DISCUSSION TITLE
Polygon 컴포넌트 MultiPolygon 지원여부

### DISCUSSION THREAD

**Discussion by zerone-jang** [zeakd/react-naver-maps#114] [Q&A] [answered]
안녕하세요.
라이브러리 잘 쓰고 있습니다!

다름이 아니라 Polygon 컴포넌트가 GeoJSON의 MultiPolygon 타입의 데이터가 적용되지 않더라구요.
혹시 MultiPolygon일 경우 Polygon 컴포넌트에 적용할 수 있는 다른 방법이 있는지 궁금합니다.

아니라면,
데이터의 타입이 Polygon 혹은 MultiPolygon일 때, 분기 처리후 MultiPolygon일 경우
여러개의 Polygon 컴포넌트를 만드는 방식으로 적용해야 할까요?!

테스트 multi-polygon json 파일 첨부합니다.
[multi-polygon.json](https://github.com/zeakd/react-naver-maps/files/14060395/multi-polygon.json)

**Comment by zeakd** [comment, marked as answer]
geoJson을 어떤식으로 적용을 하셨나요? https://navermaps.github.io/maps.js.ncp/docs/tutorial-2-DataLayer.html 네이버지도는 Multiploygon 을 Polygon으로 나누어 해석하길 기대하고있습니다.

**Reply by zerone-jang** [reply]
나누어서 Polygon 컴포넌트로 적용하니 표시 잘됐습니다.
geoJson의 이해가 부족했네요.
답변 감사합니다!
